docs/Using Fleet/Troubleshooting-live-queries.md

```diff
@@ -112,11 +112,11 @@ If the infrastructure is working correctly but the query is hanging or crashing
 
 To rule this out, you should also try out the dummy query `SELECT 1 WHERE 1 = 0;`.
 If you see "(100% responded)" with the dummy query but not with your query, then this might be an issue with:
```

Comment:
I think we'll need a change here as well, since the way this list was structured before, each bullet point was completing the sentence ("If you see '(100% responded)' with the dummy query but not with your query, then this might be an issue with the query crashing osquery on some devices...", etc.)

```suggestion
If you see "(100% responded)" with the dummy query but not with your query, then the issue might be:
```